keyboards/rart/rartlice/rules.mk

```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = STM32F103
+
+# GENERIC STM32F103C8T6 board - stm32duino bootloader
+OPT_DEFS     = -DCORTEX_VTOR_INIT=0x2000
+MCU_LDSCRIPT = STM32F103x8_stm32duino_bootloader
+BOARD        = STM32_F103_STM32DUINO
+
+DFU_ARGS        = -d 1eaf:0003 -a2 -R
+DFU_SUFFIX_ARGS = -v 1eaf -p 0003
```

Comment:
#10558

(just a note to self if your PR gets merged first)

Reply:
yes i will remove this and change to stm32Duino, waiting your PR merged